lib/utils.js

```diff
@@ -55,34 +55,68 @@ exports.splitPop = (input, splitter) => {
 };
 
 /**
- * Gets plugins from node_modules and other directories
+ * Get plugin directories from global config
  *
- * @param {object} globalConfig - Configuration object
+ * @param {object} globalConfig - global configuration file contains plugin directories
  *
- * @returns {object} - all input plugins with path
+ * @returns {array} array of directory paths
  */
-exports.getPlugins = (globalConfig) => {
-  const existing = {};
+exports.getPluginsDirs = (globalConfig) => {
```

Comment:
This file appears to be an outlier amongst our work. Would like to see these functions as `const`s that then get `module.exports`-ed at the end